### PR TITLE
Added a blog entry for 3.6.0 release

### PR DESCRIPTION
**Pull request by satishd**

### CODE REVIEW

File: blog.html
Comment (showuon) [inline blog.html:33]: This might need @mumrah 's confirmation.

File: blog.html
Comment (showuon) [inline blog.html]: <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-405%3A+Kafka+Tiered+Storage">Tiered Storage<a> is [no still] an early access feature....

File: blog.html
Comment (showuon) [inline blog.html]: ZK is marked as depracated in v3.5.0, not v3.6.0. So, maybe:
`Zookeeper is marked as deprecated. Zookeeper is planned to be removed in Apache Kafka 4.0. `

**Review by showuon — COMMENTED** [review]
Left some comments. Thanks for the PR.

File: blog.html
Comment (satishd) [reply blog.html]: Changed it to `ZooKeeper is marked as deprecated since the 3.5.0 release.`

**Comment by satishd** [comment]
Thanks @showuon for the review. Addressed them with the latest commit.

**Review by showuon — COMMENTED** [review]
Let's get this PR merged to unblock the RC build.
@mumrah can update the ZK migrating to KRaft part later.

**Review by showuon — APPROVED** [review]

**Comment by satishd** [comment]
@mumrah I am merging this PR that added a blog for 3.6.0 for now. We can have followup PRs if needed.

**Comment by mumrah** [comment]
@satishd @showuon I think we need to revert this change until the 3.6 release is complete. Here's what I just sent out on the mailing list RC0 thread:

> I noticed that the PR for the 3.6 blog post is merged. This makes the blog post live on the Kafka website https://kafka.apache.org/blog.html. The blog post (along with other public announcements) is usually the last thing we do as part of the release. I think we should probably take this down until we're done with the release, otherwise users stumbling on this post could get confused. It also contains some broken links.

**Comment by jlprat** [comment]
I just reverted it and merged it. @satishd feel free to "reopen" this one again.

**Comment by satishd** [comment]
Thanks @mumrah for clarifying the merge timeline for the blog section. I will open a PR and wait for the release candidate to be finalized before merging it. 
I will also update the wiki to make it clear that the blog should be merged only after the release candidate is finalized and it is the last thing that should be done as part of the release.

File: blog.html
Comment (todiaz) [inline blog.html:33]: non production -> non-production

**Pull request merged**
